src/control/IntersectionReferenceSystem.ts

```diff
@@ -37,11 +37,17 @@ export class IntersectionReferenceSystem {
 
   /**
    * Creates a common reference system that layers and other components can use
-   * @param path (required) array of coordinates
+   * @param path (required) array of 3d coordinates: [x, y, z]
    * @param options (optional)
    * @param options.trajectoryAngle (optional) - trajectory angle in degrees, overrides the calculated value
    */
   constructor(path: number[][], options?: ReferenceSystemOptions) {
```

Comment:
```suggestion
  constructor(path: [number,number,number][], options?: ReferenceSystemOptions) {
```
This will help typescript to only accept 3d coordinates I think

Reply:
Sure, but anytime you create an array of arrays you would have to explicitly set the dimensions or cast the data coming in from json files. We still need to validate because of javascript usage